_drafts/oryx/oryx-application-orchestration/oryx-application-orchestration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Oryx application orchestration
+description: Orchestration of the Oryx Application
+template: concept-topic-template
+---
+
+An Oryx application starts with the application orchestration. It lets you bootstrap and configure your application from reusable bits and pieces, such as the following:
+
+- [Features](./app-feature.md)
+<!-- TODO: Link to components -->
+- Components
+<!-- TODO: Link to providers -->
+- Providers
+<!-- TODO: Link to themes -->
+- Themes
+- [Resources](./resources.md)
+- [Environment](./app-environment.md)
+
+Application orchestration is used to configure and customize an Oryx applications. As Oryx is a framework, it provides different pieces of functionality for different use cases, like B2B, B2C, Back Office, or Fulfillment. And orchestration lets you select specific functionality from Oryx to match your use case.
+
+Also, application orchestration defines how functionality is loaded in an application. For example, when components are used on a page, they are lazy-loaded, but, during application startup, services are loaded eagerly.
+
+## Application builder
+
+To start using orchestration, you need to import [`appBuilder`](./app-builder.md) from `@spryker-oryx/core`. Then, you can add functionality, like features and theme, to your application.
```

Comment:
```suggestion
To start using orchestration, you need to import [`appBuilder`](./app-builder.md) from `@spryker-oryx/core`. Then, you can add functionality, like features and themes, to your application.
```

Reply:
@gund Is it usually one or multiple themes defined for an app?